### ISSUE TITLE
[studio] Changing the default encryption key and salt breaks studio encryption tool 

### ISSUE THREAD

**Issue by cortiz** [craftercms/craftercms#4170]
## Describe the bug
Changing the default encryption key and salt breaks studio encryption tool, 

If a user changes the variables CRAFTER_ENCRYPTION_KEY and CRAFTER_ENCRYPTION_SALT 
The [encryption tool]((https://docs.craftercms.org/en/3.1/system-administrators/activities/authoring/main-menu-encryption-tool.html?highlight=encrytion)) no longer produces valid results. 
This is most likely, due that this setting is not passed to the underlying studio's engine config and uses the default values to encrypt but the key used by Studio is not the same as engine.

## To Reproduce
1. Set CRAFTER_ENCRYPTION_KEY and CRAFTER_ENCRYPTION_SALT  as environment variables and set them any valid value
2. In the same shell start Crafter Studio
3. Use the encryption tool, and set any value 
4. In the Engine Site config, add a key for example 
```xml
<site> 
...
     <test>${enc:LmMrFnAW+r4g3IZ8Wzl3QzHZytlZdjYa7eBC2Tm01Bo=}</test>
...
</site>
```
5. Try to use the value in any valid place like 
```groovy
def sample = siteConfig.getString("test")
return ["sample":sample]
```

## Expected behavior
1. The actual value should return instead of the hash value
2. The studio tool should use the same values set in CRAFTER_ENCRYPTION_KEY and CRAFTER_ENCRYPTION_SALT and not the default ones.
### Version
```
Studio Version Number: 3.1.5-fb3509

Build Number: fb35092a40f7b3d5edf1f2ef61e437b61cc2e5e6

Build Date/Time: 02-21-2020 17:17:38 -0600
```
### OS
Linux (Crafter CMS AIM)

### Browser
Any

## Additional context
None

**Comment by sumerjabri** [comment]
Fixed in 3.1.8+

**Comment by sumerjabri** [comment]
@cortiz might also be fixed in 3.1.6. Regardless, you must upgrade to 3.1.7+ for security patches.
